src/components/selector/index.tsx

```diff
@@ -77,12 +87,15 @@ export class Selector {
 	}
 	private move(direction: -1 | 0 | 1): void {
 		if (direction) {
-			let selectedIndex = this.items.findIndex(item => item == this.selectedElement)
+			let markedIndex = this.items.findIndex(item => item.marked)
+			if (markedIndex == -1)
+				markedIndex = this.items.findIndex(item => item.selected)
+			if (this.items[markedIndex])
+				this.items[markedIndex].marked = false
```

Comment:
i dont get this

Reply:
if there is something already marked, i.e. not moving the first time since opening, the previous will be "unmarked"